Translation/en/Text/Main/Menu-GirlsRoom.txt

```diff
@@ -60,9 +60,17 @@
 滞在先の変更=Change Location
 滞在を解除=Check-out
 
+//Change Girl's Location
+移動中=Moving
+ここに移動=Go Here
+入れ替える=Replace
+
 プレイヤーの愛称=Player's Nickname
 変更=Change
 
+全解除=All
```

Comment:
Cancel All

Reply:
Hmm... This one clear all traits when you select a girl on the room, so I think that "Clear All" is better